tron/utils/timeutils.py

```diff
@@ -76,6 +77,9 @@ class DateArithmetic(object):
         'day': '%d',
         'hour': '%H',
         'shortdate': '%Y-%m-%d',
+        'ymd': '%Y%m%d',
+        'ymdh': '%Y%m%d%H',
+        'ymdhm': '%Y%m%d%H%M',
```

Comment:
Are these strings are going to be used by humans? If so, can we make them more readable with separators, e.g. %Y-%m-%d:%H:%M ?